windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -28,9 +28,24 @@ WMCO supports self-managed clusters built using installer-provisioned infrastruc
 * Microsoft Azure
 * VMware vSphere
 
-The following Windows Server operating systems are supported in this release of the WMCO:
+The following Windows Server operating systems are supported in this release of the WMCO, depending on the platform to which your cluster is installed:
```

Comment:
s/depending on the platform to which your cluster is installed:/depending on which platform your cluster is installed on: